assets/app/views/_pod-template.html

```diff
@@ -48,6 +48,13 @@
     </div>
 
     <div ng-if="container.ports.length > 0" class="pod-template-ports">
+      <div>
+        <span ng-show="container.jolokiaUrl" class="pull-right">
```

Comment:
I thought we were going to inject the link inside the pod template itself using an extension point